GameServerCore/Domain/GameObjects/ITarget.cs

```diff
@@ -1,9 +1,12 @@
-﻿namespace GameServerCore.Domain.GameObjects
+﻿using System.Numerics;
+
+namespace GameServerCore.Domain.GameObjects
 {
     public interface ITarget
     {
         float X { get; }
         float Y { get; }
         bool IsSimpleTarget { get; }
+        Vector2 GetPosition();
```

Comment:
I'd also like to wait for @danil179 or @moonshadow565's comment on this addition, as they've been more in charge of the refactor regarding `GameServerCore`.

To me this looks like:
- The position could be derived from the `X` and `Y` fields that are already in the interface
- If we add a vector to the interface, it could be a property instead of a getter method
- If we add a vector for the position to the interface, that obsoletes the X and Y floats from the interface

Reply:
Yeah wasnt quite sure, but the method exists allready in the Target so i thought i add it to the interface. Its not necessarily needed though.

Reply:
Yeah, it's a bit out of scope of this PR in that case.

The `GetPosition` function already being implemented sounds a bit odd to me, because as said it's redundant to support both Vector2 and non-Vector2 positions. Let's wait for more comment on the subject, or create a new issue about it and figure it out later if there's no resolution shortly.

Reply:
We need to change all X&Y positions into objects like `Vector2D`, it should be done by refactoring the whole project, many classes and interfaces implements ITarget (because IGameObject use it).

I recommend to open issue about that and refactor the project later, now just make GetPosition to return X&Y positions variables I think, that way you don't need to refactor the whole project.

Reply:
Created issue #721